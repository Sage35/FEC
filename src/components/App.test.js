// this is where Tests for App.jsx go!
const React = require('react');
const enzyme = require('enzyme');
const Adapter = require('enzyme-adapter-react-16');

enzyme.configure({ adapter: new Adapter() });

const App = require('./App').default;

// example test using shallow render
describe('App', () => {
  test('should render App with correct text', () => {
    const wrapper = enzyme.shallow(<App />);
    expect(wrapper.exists()).toBeTruthy();
    expect(wrapper.text() === 'Hello World!').toBeTruthy;
  });
});
