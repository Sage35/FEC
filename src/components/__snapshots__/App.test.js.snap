// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App Snapshot 1`] = `
<Fragment>
  <SocialMediaIcons />
  <div
    id="overall-container"
  >
    <Connect(Overview) />
    <Connect(RelatedItems)
      async={true}
    />
    <Connect(Reviews) />
  </div>
</Fragment>
`;
